custom-vu/src/main/kotlin/jces1209/vu/page/DcCommenting.kt

```diff
@@ -28,4 +31,20 @@ class DcCommenting(
     override fun waitForTheNewComment() {
         driver.wait(visibilityOfElementLocated(By.cssSelector(".activity-comment.focused")))
     }
+
+    override fun mentionUser() {
```

Comment:
I had an old branch with mentioning: https://github.com/atlassian/jces-1209/pull/9
I think my old impl used fewer locators and more stable ones.